packages/web-shared/components/FeatureFeed/Features/HeroListFeature.js

```diff
@@ -115,9 +116,9 @@ function HeroListFeature(props = {}) {
               title="Watch now"
               onClick={handleWatchNowPress}
             />
-            {props?.feature?.primaryAction?.relatedNode !== null ? (
+            {props?.feature?.primaryAction?.relatedNode ? (
```

Comment:
If this is an empty object won't this still render? I think I ran into that issue recently, but I can't remember why. 

Actually, I think this is correct. relatedNode shouldn't return an empty {}. So, that would be a backend issue I believe.

Reply:
@jordanwade  Oh, yeah, I was under the assumption that it would either return an object with values or `undefined`. I forgot about the case with empty state. 

If you think it shouldn't return an empty {}, then should be good to go